docs/en/latest/plugins/kafka-logger.md

```diff
@@ -48,6 +48,7 @@ For more info on Batch-Processor in Apache APISIX please refer.
 | broker_list      | object  | required    |                |         | An array of Kafka brokers.                                                               |
 | kafka_topic      | string  | required    |                |         | Target  topic to push data.                                                              |
 | producer_type    | string  | optional    | async          | ["async", "sync"]        | Producer's mode of sending messages.          |
+| required_acks          | integer | optional    | 1              | [0, 1, -1] | The number of acknowledgments the producer requires the leader to have received before considering a request complete. This controls the durability of records that are sent. Semantics is the same as kafka producer acks.        |
```

Comment:
Could you also explain the specific meaning of the individual options?

Reply:
docs updated~